extensions/jump/README.md

```diff
@@ -0,0 +1,15 @@
+# Jump
+
+Jump to websites, folders, or files
```

Comment:
I think most of the users (including myself) would struggle to understand how it works, I saw your demo, and it works as intended, but where do we get the data from, and how to generate it? 

Might be good to describe it in the README so it's possible to use 😊

Reply:
I've updated the README to describe the functionality better, what do you think of it now?